examples/csrf/samesite/docker-compose.yml

```diff
@@ -1,4 +1,4 @@
-version: '2'
+version: '3.7'
```

Comment:
can i suggest making this change in a separate PR

Reply:
Sure!

Reply:
@baremaximum - i think this is a good change - but we probs want to check other files etc - please do PR if you get chance

Reply:
@phlax Done! :)